lib/modules/showImages.js

```diff
@@ -908,6 +908,16 @@ function shouldLinkBeDeferred(element) {
 const inText = element => !!element.closest('.md, .search-result-footer');
 
 async function checkElementForMedia(element) {
+	// Prevents relatively addressed anchor link urls from refreshing the page,
+	// while also opening all #res:settings urls on the current page
+	const hash = (/^#.*|#res:settings.*/gi).exec(element.getAttribute('href'));
+	if (hash) {
+		element.addEventListener('click', e => {
+			e.preventDefault();
```

Comment:
If the user middle-clicks, right-clicks, ctrl-clicks, etc. this function shouldn't run.

Reply:
You're right, the ctrl-click would run, incorrectly, I'll fix that. Middle and Right clicks won't fire the event though, so those will work.